### ISSUE TITLE
GitHub warning about security vulnerability

### ISSUE THREAD

**Issue by annagree** [Sage-Bionetworks/Agora#433]
Current security vulnerability flagged by GitHub:
https://github.com/Sage-Bionetworks/Agora/network/dependencies

<img width="1151" alt="screen shot 2018-10-03 at 11 52 14 am" src="https://user-images.githubusercontent.com/40642407/46432353-f0e1ec00-c702-11e8-995e-eeb41c1f2dbd.png">

**Comment by kdaily** [comment]
https://github.com/Sage-Bionetworks/Agora/network/dependencies

The vulnerability is in `bootstrap 4.0.0`. Reccommended upgrade to `bootstrap 4.1.2`.

**Comment by Zaknarfen** [comment]
I was seeing this for the `karma-webpack` package also. I'll update asap.

**Comment by Zaknarfen** [comment]
```
                       === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Low             Regular Expression Denial of Service

  Package         debug

  Patched in      >= 2.6.9 < 3.0.0 || >= 3.1.0

  Dependency of   @compodoc/compodoc [dev]

  Path            @compodoc/compodoc > live-server > connect > debug

  More info       https://nodesecurity.io/advisories/534


  Low             Regular Expression Denial of Service

  Package         debug

  Patched in      >= 2.6.9 < 3.0.0 || >= 3.1.0

  Dependency of   @compodoc/compodoc [dev]

  Path            @compodoc/compodoc > live-server > connect > finalhandler >
                  debug

  More info       https://nodesecurity.io/advisories/534
```

**Comment by Zaknarfen** [comment]
I updated what was possible, but for `@compodoc/compodoc` we still need to keep an eye for [this](https://github.com/compodoc/compodoc/issues/665) issue.
